app/models/mixins/filterable_mixin.rb

```diff
@@ -9,10 +9,11 @@ def find_filtered(number, options = {})
       options[:include] = get_include_for_find(options[:include]) unless options.delete(:eager_loading) == false
       if mfilters.blank?
         # do normal find if no filters
-        result = find(number, :conditions => options[:conditions],
-          :order => options[:order_by] || options[:order],
-          :joins => options[:join] || options[:joins], :include => options[:include],
-          :select => options[:select])
+        result = where(options[:conditions])
+                 .order(options[:order])
+                 .joins(options[:join] || options[:joins])
+                 .includes(options[:include])
+                 .select(options[:select])
```

Comment:
@matthewd do you have a major preference on going to legacy finders here?
For some reason I thought you stated the desire. Although, I don't see any comments from you. I was probably imagining this.
While I prefer using legacy finders, I'm unsure what that would introduce and whether it handles `:join` vs `:joins`.

So I'm leaning towards :shipit: